juriscraper/opinions/united_states/state/sd.py

```diff
@@ -3,106 +3,40 @@
 # - 2013-06-11: Birth.
 # - 2013-08-06: Revised by Brian Carver
 # - 2014-08-05: Updated URL by mlr
+# - 2021-12-28: Updated by flooie
 
 import re
 
-from lxml import html
+from juriscraper.lib.html_utils import get_row_column_text
+from juriscraper.lib.string_utils import titlecase
+from juriscraper.OpinionSiteLinear import OpinionSiteLinear
 
-from juriscraper.AbstractSite import logger
-from juriscraper.lib.html_utils import fix_links_in_lxml_tree
-from juriscraper.lib.string_utils import convert_date_string, titlecase
-from juriscraper.OpinionSiteWebDriven import OpinionSiteWebDriven
 
-
-class Site(OpinionSiteWebDriven):
+class Site(OpinionSiteLinear):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.court_id = self.__module__
+        self.status = "Published"
```

Comment:
Hm, is this being used somewhere? I think this attribute isn't ever used?

Reply:
yes - status is required on all scrapers.  This just sets status to published for the SD supremes